supportbot/supportbot/utils/uisp_data.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import json
+from dotenv import load_dotenv
+import os
+import re
+from datetime import datetime
+
+load_dotenv()
+
+def get_uisp_devices():
+    response = requests.get("https://uisp.mesh.nycmesh.net/nms/api/v2.1/devices", headers={'x-auth-token': os.environ.get('NYCMESH_TOOL_AUTH_TOKEN')}, verify=False)
+
```

Comment:
nit: Might be good to pull this URL out into a separate constants file?

Reply:
Oh good idea